doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc

```diff
@@ -65,6 +65,12 @@ If you configure an external Maven repository (`MAVEN_REPO_URL`), change `MAVEN_
 +
 If your authoring environment uses a built-in {CENTRAL} Maven repository, change `MAVEN_MIRROR_OF` to exclude the artifacts in this repository from the mirror: `external:*,!repo-{PRODUCT_INIT}centr`.
 +
+** In some cases, you might want to persist the Maven repository cache for {CENTRAL}. By default, the cache is not persisted, so when you restart or scale a {CENTRAL} pod, all Maven artifacts are downloaded again and all projects within {CENTRAL} must be built again. If you enable persistence for the cache, the download is not necessary and startup time can improve in some situations. However, significant additional space on the {CENTRAL} persistence volume is required.
++
+To enable persistence for the Maven repository cache, set the `KIE_PERSIST_MAVEN_REPO` environment variable to `true`.
++
+If you set `KIE_PERSIST_MAVEN_REPO` to `true`, you can optionally set a custom path for the cache using the `KIE_M2_REPO_DIR` variable. The default path is `/opt/kie/data/m2`. Files in the `/opt/kie/data` directory tree are persisted.
```

Comment:
```suggestion
If you set `KIE_PERSIST_MAVEN_REPO` to `true`, you can optionally set a custom path for the cache using the `KIE_M2_REPO_DIR` variable. Make sure that your configured path is persisted and have created it's own Persistent Volume. The default path is `/opt/kie/data/m2`. Files in the `/opt/kie/data` directory tree are persisted in default rhpamcentr-claim.
```

Reply:
New issue opened for BC PV size configuration https://issues.redhat.com/browse/RHPAM-3320